### PR TITLE
DPTS-459 Create base infrastructure

### PR DESCRIPTION
**Pull request by dehe1**
Create the underlying framework for the housekeeping jobs: infrastructure, pipeline, docker

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -19,122 +34,98 @@ test: # Test project
...
-	[ $$(make project-branch-perf-test) != true ] && exit 0
-	echo TODO: $(@)
+# --------------------------------------
 
-run-security-test:
-	[ $$(make project-branch-sec-test) != true ] && exit 0
-	echo TODO: $(@)
+lambda-alias: ### Updates new lambda version with alias based on commit hash - Mandatory PROFILE=[profile]
+	eval "$$(make aws-assume-role-export-variables)"
+	function=$(SERVICE_PREFIX)-rd-lambda
```
Comment (timrickwood) [inline Makefile:77]: rd ? reference to road distance ?

Comment (dehe1) [reply Makefile]: Thanks, refactored and removed

**Review by timrickwood — APPROVED** [review]

File: build/automation/var/profile/nonprod.mk [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+# Infrastructure variables
+
+STACKS := secrets,s3,security-groups,filter
+TASKS := referralroles
+ENVIRONMENT_LIST := ["test","test1","test2","test3","test4","fix","performance","regression"]
+
+TF_VAR_image_version := $(or $(BUILD_TAG), latest)
+TF_VAR_s3_tf_state_key := $(PROJECT_ID)/$(ENV)/s3/terraform.state
+TF_VAR_security_groups_tf_state_key := $(PROJECT_ID)/$(ENV)/security-groups/terraform.state
+TF_VAR_core_dos_db_sg := sg-05fdf44634b750fbd
```
Comment (tamaragoldschmidt) [inline build/automation/var/profile/nonprod.mk:22]: Should this SG be a variable?

Comment (tamaragoldschmidt) [reply build/automation/var/profile/nonprod.mk:22]: discussed with Dec

**Review by tamaragoldschmidt — APPROVED** [review]

**Pull request merged**